app/javascript/react/components/molecules/Calendar/atoms/Month/Month.style.tsx

```diff
@@ -14,24 +14,25 @@ const Week = styled.div`
 const Month = styled.div`
   display: flex;
   flex-direction: column;
+  width: 100%;
 `;
 
 const MonthName = styled.span`
   font-size: 24px;
```

Comment:
I saw that for mobile it's 16 px, capital and mora "air" between letters, letter-spacing of 6px.

Reply:
Done